app/src/org/commcare/activities/CommCareSetupActivity.java

```diff
@@ -637,13 +637,21 @@ public boolean onOptionsItemSelected(MenuItem item) {
     /**
      * Return to or launch dispatch activity.
      *
-     * @param failed         did installation occur successfully?
+     * @param failed did installation occur successfully?
      */
     private void done(boolean failed) {
         if (Intent.ACTION_VIEW.equals(CommCareSetupActivity.this.getIntent().getAction())) {
-            //Call out to CommCare Home
-            Intent i = new Intent(getApplicationContext(), DispatchActivity.class);
-            startActivity(i);
+            // app installed from external action
+            if (getIntent().getBooleanExtra(FORCE_VALIDATE_KEY, false)) {
+                // force multimedia validation to ensure app shows up in multiple apps list

```

Comment:
I'm confused by this. Why isn't the usual mm verification path sufficient here? And why is it saying that this came from the app manager when it didn't?